tests/unit/io/expected_recipe.yml

```diff
@@ -11,426 +11,434 @@ Kinematics:
-                        FinalState:
+                        helicity: -0.0
+                      - particle: pi0
+                        final_state_ids:
                           - 4
-                        Helicity: -0.0
-                    RecoilSystem:
-                      RecoilFinalState:
+                        helicity: -0.0
+                    recoil_system:
+                      recoil_final_state:
                         - 2
 
-ParticleList:
-  J/psi(1S):
-    PID: 443
-    Mass: 3.0969
-    Width: 9.29e-05
-    QuantumNumbers:
-      Spin: 1
-      Charge: 0
-      Parity: -1
-      CParity: -1
-      GParity: -1
-      IsoSpin: 0
-  gamma:
-    PID: 22
-    Mass: 0.0
-    QuantumNumbers:
-      Spin: 1
-      Charge: 0
-      Parity: -1
-      CParity: -1
-  pi0:
-    PID: 111
-    Mass: 0.1349768
-    Width: 7.73e-09
-    QuantumNumbers:
-      Spin: 0
-      Charge: 0
-      Parity: -1
-      CParity: 1
-      GParity: -1
-      IsoSpin:
-        Value: 1
-        Projection: 0
-  f(0)(980):
-    PID: 9010221
-    Mass: 0.99
-    Width: 0.06
-    QuantumNumbers:
-      Spin: 0
-      Charge: 0
-      Parity: 1
-      CParity: 1
-      GParity: 1
-      IsoSpin: 0
-  f(0)(1500):
-    PID: 9030221
-    Mass: 1.506
-    Width: 0.112
-    QuantumNumbers:
-      Spin: 0
-      Charge: 0
-      Parity: 1
-      CParity: 1
-      GParity: 1
-      IsoSpin: 0
+particles:
+  - name: J/psi(1S)
+    pid: 443
+    mass: 3.0969
+    width: 9.29e-05
+    spin: 1.0
+    parity:
+      value: -1
+    c_parity:
+      value: -1
+    g_parity:
+      value: -1
+    isospin:
+      magnitude: 0.0
+      projection: 0.0
+  - name: gamma
+    pid: 22
+    mass: 0.0
+    spin: 1.0
+    parity:
+      value: -1
+    c_parity:
+      value: -1
+  - name: pi0
+    pid: 111
+    mass: 0.1349768
+    width: 7.73e-09
+    spin: 0.0
+    parity:
+      value: -1
+    c_parity:
+      value: 1
+    g_parity:
+      value: -1
+    isospin:
+      magnitude: 1.0
+      projection: 0.0
+  - name: f(0)(980)
+    pid: 9010221
+    mass: 0.99
+    width: 0.06
+    spin: 0.0
+    parity:
+      value: 1
+    c_parity:
+      value: 1
+    g_parity:
+      value: 1
+    isospin:
+      magnitude: 0.0
+      projection: 0.0
+  - name: f(0)(1500)
+    pid: 9030221
+    mass: 1.506
+    width: 0.112
+    spin: 0.0
+    parity:
+      value: 1
+    c_parity:
+      value: 1
+    g_parity:
+      value: 1
+    isospin:
+      magnitude: 0.0
+      projection: 0.0
 
-Dynamics:
+dynamics:
   J/psi(1S):
-    Type: NonDynamic
-    FormFactor:
-      Type: BlattWeisskopf
-      MesonRadius: *par5
+    type: NonDynamic
+    form_factor:
+      type: BlattWeisskopf
+      meson_radius: *par5
   f(0)(980):
-    Type: RelativisticBreitWigner
-    PoleParameters:
-      Real: *par6
-      Imaginary: *par7
-    FormFactor:
-      Type: BlattWeisskopf
-      MesonRadius: *par8
+    type: RelativisticBreitWigner
+    pole_position: *par6
+    pole_width: *par7
```

Comment:
We should really define a better name for the pole. The pole position in literature is the complex number, which defines the pole in the complex energy plane. Maybe we should name it `pole_real`, `pole_imag`.